### PR TITLE
Add Maldives holidays

### PR DESCRIPTION
**Pull request by kasya**
## Proposed change

Initial support for Maldives holidays.

## Type of change

- [x] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've added references to all holidays information sources used in this PR
- [x] The code style looks good: `make pre-commit` command generates no changes
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

**Review by KJhellico — COMMENTED** [review]
@kasya, thank you for new country! 👍 Please look at few suggestions.

File: tests/countries/test_maldives.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            ("2020-07-31", "Eid al-Adha* (*estimated)"),
+            ("2020-11-03", "Victory Day"),
+            ("2020-05-26", "Eid al-Fitr* (*estimated)"),
+            ("2020-11-11", "Republic Day"),
+            ("2020-07-30", "Hajj Day* (*estimated)"),
+            ("2020-08-03", "Eid al-Adha* (*estimated)"),
+            ("2020-11-16", "The Day Maldives Embraced Islam* (*estimated)"),
+            ("2020-10-29", "Mawlid al-Nabi* (*estimated)"),
+            ("2020-05-01", "Labor Day"),
+            ("2020-05-24", "Eid al-Fitr* (*estimated)"),
```
Comment (arkid15r) [inline tests/countries/test_maldives.py:82]: Could you sort this too?

**Review by arkid15r — COMMENTED** [review]
It's great to see another country support added to the library.
Nicely done, @kasya!

**Review by KJhellico — APPROVED** [review]
LGTM.

**Pull request merged**